src/app/pages/ArticlePage/Byline/index.tsx

```diff
@@ -0,0 +1,94 @@
+/* eslint-disable jsx-a11y/aria-role */
+import React, { useContext, ReactNode } from 'react';
+import pathOr from 'ramda/src/pathOr';
+import { ServiceContext } from '../../../contexts/ServiceContext';
+import VisuallyHiddenText from '../../../legacy/psammead/psammead-visually-hidden-text/src';
+import { Author, JobRole, BylineList, LineBreak } from './index.styles';
+
+type Props = {
+  blocks: any;
+  children?: ReactNode;
```

Comment:
Similar to above, `PropsWithChildren` could be wrapped around your own type definition.

Reply:
fixed 👍